precog/miners/base_miner.py

```diff
@@ -23,11 +22,12 @@ def get_point_estimate(timestamp: str) -> float:
 
     # Set the time range to be as small as possible for query speed
     # Set the start time as 2 seconds prior to the provided time
-    start_time: str = datetime_to_CM_timestamp(iso8601_to_datetime(timestamp) - timedelta(days=1))
-    end_time: str = datetime_to_CM_timestamp(iso8601_to_datetime(timestamp))  # built-ins handle CM API's formatting
+    # Ensure both timestamps are correctly typed and set to UTC
+    start_time = get_before(timestamp=timestamp, days=0, minutes=0, seconds=2)
```

Comment:
@umocm what do you think now?

Reply:
looks much cleaner to me!  Please refer to our chat to see if we would rather use **kwargs limit_per_asset and paging_from end to for this specific purpose